running.md

```diff
@@ -0,0 +1,141 @@
+## Debian Package Install
+
+`sogs-standalone` package runs sogs bare and it's the fastest and simplest setup (recommended).
+
+`sogs-proxied` package runs sogs behind an nginx reverse proxy and allows for more advance flexible 
+configuration, for experienced admins only.
+
+To install the debian package for sogs (`sogs-standalone` or `sogs-proxied` ) do the following:
+
+```
+sudo curl -so /etc/apt/trusted.gpg.d/oxen.gpg https://deb.oxen.io/pub.gpg
+echo "deb https://deb.oxen.io $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/oxen.list
+sudo apt update
+sudo apt install sogs-standalone
+```
+
+It will prompt for server name on install.
+
+Add a room to the server:
+
+```bash
+sogs --add-room room_name_here
```

Comment:
```
sogs --add-room TOKEN --name "NAME"
```
(Replace TOKEN with the token to be used in the session URL; replace NAME with the room name to be displayed in Session).